src/GitHub.Exports/Services/IGitHubContextService.cs

```diff
@@ -30,14 +30,14 @@ public interface IGitHubContextService
         /// <summary>
         /// Convert a context to a repository URL.
         /// </summary>
-        /// <param name="context">The context to convert.</param>
+        /// <param name="windowTitle">A browser window title.</param>
         /// <returns>A repository URL</returns>
         GitHubContext FindContextFromWindowTitle(string windowTitle);
 
         /// <summary>
         /// Find a context from a browser window title.
         /// </summary>
-        /// <param name="windowTitle">A browser window title.</param>
+        /// <param name="context"></param>
```

Comment:
`The context`? Not very descriptive. Do we need more info than this? @jcansdale?